### PR TITLE
LPS-130387 Search on properties field: eliminate wildcard usage and use a different type of query

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://issues.liferay.com/browse/LPS-130387

The properties field is a field that mainly integrates the title and the description, and it insert extra-attributes only if the file has it and in my opinion is without use.

since the properties field is inserted on the query as a wildcard with the double star and we are not effectively using it. I'm deleting it from the search of the documents and media.

You can check the deletion of it on the search field on : 

Adding the search bar on the page
Adding the Search Insights on the page

image

making a search and seeing the result query

Thanks

### CODE REVIEW

File: portal-impl/src/com/liferay/portlet/documentlibrary/model/impl/DLFileEntryImpl.java [hunk 1]
```diff
@@ -245,6 +245,10 @@ public Lock getLock() {
 			DLFileEntry.class.getName(), getFileEntryId());
 	}
 
+	/**
```
Comment (AliciaGarciaGarcia) [inline portal-impl/src/com/liferay/portlet/documentlibrary/model/impl/DLFileEntryImpl.java:248]: @adolfopa I decided to deprecate it cause is only used on the contributor.
but I have not a strong opinion about it :D

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**